NEWS.md

```diff
@@ -11,6 +11,8 @@
   - Improved logging of what guider was used when using NotWorn and optional backup guider, #1156
 
   - Skip night in part 4 csv report if guider was NotWorn, #1156
+  
+- Part 5: Revised rounding of seconds in timestamps to match epoch size used in part 5 when adding the sleep period time windows to the time series (g.part5.wakesleepwindows). #1192
```

Comment:
To me this message and the matching issue item are not clear. It needs to be possible for a reader to understand how severe the problem is and how it may have affected their data. It sounds like this affect every timestamp with a seconds above 30, because on average half of the timestamps have seconds above 30.

Could you please update the issue item with more details and rephrase this NEWS item to be good summary of that item.

Reply:
Thanks, for expanding the issue message. With more details now available there, the NEWS item can be short. I have just committed a revisions to the NEWS and will merge this PR when the tests have all passed.